jupyterhub/tests/test_api.py

```diff
@@ -176,7 +230,7 @@ def fill_user(model):
     users = [normalize_user(u) for u in users]
     assert users == [
         fill_user({'name': 'admin', 'admin': True}),
-        fill_user({'name': 'user', 'admin': False, 'last_activity': None}),
```

Comment:
Why this change?

Reply:
When I added the test above the `last_activity` field on this `user` server was no longer None, I'm not entirely sure why. I assumed it was because as a result of the new test that runs before this (and these tests aren't idempotent) that something _touched_ the server so it had an activity set, but I'm not familiar enough with the code to know what might cause that side effect.